updates_and_signals/atomic_message_handlers/starter.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+import asyncio
+import logging
+import uuid
+from typing import Optional
+
+from temporalio import client, common
+from temporalio.client import Client, WorkflowHandle
+
+from updates_and_signals.atomic_message_handlers.workflow import (
+    ClusterManagerInput,
+    ClusterManagerWorkflow,
+)
+
+
+async def do_cluster_lifecycle(wf: WorkflowHandle, delay_seconds: Optional[int] = None):
+    allocation_updates = []
+    for i in range(6):
+        allocation_updates.append(
+            wf.execute_update(
+                ClusterManagerWorkflow.allocate_n_nodes_to_job, args=[f"task-{i}", 2]
```

Comment:
I think we want to discourage multiple arguments to things (workflows, activities, signals, queries, updates, etc)